README.md

```diff
@@ -931,17 +924,27 @@ guard let thingThree = thingThree else {
 }
 ```
 
-* **3.11.7** Don’t use one-liners for `guard` statements.
+* **3.11.7** For single-statement returning guards, prefer to maintain the return statement inline.
 
 
 ```swift
 // PREFERRED
-guard let thingOne = thingOne else {
-    return
+guard let thingOne = thingOne else { return nil }
+
+guard let name = json["name"] as? String,
+  let coordinatesJSON = json["coordinates"] as? [String: Double],
+  let latitude = coordinatesJSON["lat"],
+  let longitude = coordinatesJSON["lng"],
+  let mealsJSON = json["meals"] as? [String] else { return }
```

Comment:
Hmm I am ok with one liner guards but when you have multiple properties, I prefer the return to be on the second line i.e. 
```
guard let name = json["name"] as? String,
  let coordinatesJSON = json["coordinates"] as? [String: Double],
  let latitude = coordinatesJSON["lat"],
  let longitude = coordinatesJSON["lng"],
  let mealsJSON = json["meals"] as? [String] else
 {
    return 
 }

Reply:
Ya, that's valid point. Not my preference but I see where you're coming from. I'll take that back to the team so we can come with something that's agreed by the majority.

Reply:
100% with Miwand, any multi show following with a new line

Reply:
Looks like this is what the people want. :)
Change on [52319ab](https://github.com/ecobee/swift-style-guide/pull/3/commits/52319ab2b7af77e6ce19041343285762652a735e).